src/common/models/user_sets.ts

```diff
@@ -2,34 +2,39 @@
  * This defines UserSets and MergedUserSets
  */
 
-import { Model } from '.';
+import { Dictionary } from 'express-serve-static-core';
+import { generic, Model } from '.';
```

Comment:
```suggestion
import { Model } from '.';
```
`.toObject` later will receive proper typing automatically.

Reply:
A vscode auto-filling imports sometimes picks up the wrong one.